src/index.js

```diff
@@ -1 +1,14 @@
+import './index.html';
 import './style.css';
+import display from './modules/displayList.js';
+import Meal from './modules/meal-Class.js';
+import menuCounter from './modules/itemCounter.js';
+import { likes, displayLikes } from './modules/likesCounter.js';
+import './assets/close.svg';
+
+display();
+likes();
+displayLikes();
+menuCounter();
+const niceMeal = new Meal();
+niceMeal.getMeal();
```

Comment:
## Before
![image](https://user-images.githubusercontent.com/93039359/225808415-0f3801ad-2177-4f88-a253-9d550d6f0010.png)

## After
![image](https://user-images.githubusercontent.com/93039359/225808276-77677d34-45d3-46ea-98a8-e1e069b1dd17.png)

## [OPTIONAL]
- [ ] Kudos on implementing the async await 👏 but I guess while updating the `displayLikes()` method you broke something in your application 😨 When I reload your page your application won't get likes but when I like any recipe then your app gets like for only that recipe 🤔 Kindly fix it 😊

You can fix it by calling the `displayLIkes()` method at the end of your index.js file 😊 like below

![image](https://user-images.githubusercontent.com/93039359/225813167-50dc8e2d-3098-4bec-a27c-e1a489e2d8a7.png)


Now you are thinking I should call it once and I should remove the callback at line 11 but if you do it then it won't work. You have to keep your code same and call the displayLikes() method at the end 😊

Reply:
thanks